src/components/ProfitAndLossChart/ProfitAndLossChart.tsx

```diff
@@ -591,6 +606,38 @@ export const ProfitAndLossChart = () => {
           xAxisId='expenses'
           stackId='expenses'
         />
+        <Bar
```

Comment:
After looking at the example image you posted in the PR description (very helpful), I think my preference would be to put the `uncategorizedOutflowsInverse` *above* the categorized expenses and categorized revenue. I think it feels better to have the profit dot visibly sit on top of the `categorized` bars and then have the uncategorized bars higher up.

Reply:
I've changed the order, screenshot attached to the previous comment